### ISSUE TITLE
Middle mouse button does not work in the tree of elements (resources and files)

### ISSUE THREAD

**Issue by Ruslan-Aleev** [modxcms/revolution#14059]
This problem has been discussed earlier:
https://github.com/modxcms/revolution/issues/12374
https://github.com/modxcms/revolution/pull/13103

But the **mouse does not work** in the browser, although **"ctrl + click" - works**.

**In Windos / Linux OS the middle button click event does not hold the proper information in the onclick event, but it does in onmouseup.**

p.s. Without the middle mouse button - as if the finger was cut off, especially if it's standard browser behavior :)

MODX 2.6.5
Google Chrome 68.0.3440.106
Firefox 61.0.2
Probably other browsers

**Comment by JoshuaLuckers** [comment]
Relevant information can be found in the comments of PR #14061: https://github.com/modxcms/revolution/pull/14061#issuecomment-416001521

**Comment by Ruslan-Aleev** [comment]
@JoshuaLuckers There is information about this bug?
The question is important :), periodically messages turn up on the network about it, Windows users are many times more than the rest of the OS, and without the middle mouse button the interaction with MODX is getting worse.

**Comment by JoshuaLuckers** [comment]
I am not able to reproduce the issue. The combination of your OS and/or mouse (might be the driver) are not sending the right event. Since I'm unable to reproduce it I can not fix it myself.

Have you tried it with another mouse on another machine?

**Comment by Ruslan-Aleev** [comment]
@JoshuaLuckers Yes, I tested on different devices and OS, I found out that on Windows (I tested on 7 and 10), on Linux (Mint) the wheel does not work at all (the current window opens in a new tab).
It only works on Mac OS and does not work completely, for example, the "+" icon next to the category, for example, chunks, also does not open a new window.

**Comment by JoshuaLuckers** [comment]
Thanks for testing. Did you test with the same mouse or with different ones?

**Comment by Ruslan-Aleev** [comment]
Tested on different mouse, computers and OS, and several people helped in the test with own mouse :)

**Comment by JoshuaLuckers** [comment]
Are you available on Slack? I want you to test something if that is possible.

**Comment by Ruslan-Aleev** [comment]
@JoshuaLuckers Yes, I am in slack - Ruslan Aleev, including https://modxcommunity.slack.com

**Comment by JoshuaLuckers** [comment]
Fixed by PR #14243
